File: OOP/StaticMethods.py
class MathUtils:
    PI = 3.14159

    @staticmethod
    def add(a, b):
        return a + b

    @staticmethod
    def multiply(a, b):
        return a * b

    @staticmethod
    def circle_area(r):
        return MathUtils.PI * r * r


class Counter:
    _count = 0

    def __init__(self):
        Counter._count += 1

    @staticmethod
    def get_count():
        return Counter._count


print(MathUtils.add(3, 4))
print(MathUtils.circle_area(5))

Counter()
Counter()
Counter()
print(Counter.get_count())

File: OOP/StaticMethods.js
class MathUtils {
  static PI = 3.14159;

  static add(a, b) { return a + b; }
  static multiply(a, b) { return a * b; }
  static circleArea(r) { return MathUtils.PI * r * r; }
}

class Counter {
  static #count = 0;
  constructor() { Counter.#count++; }
  static getCount() { return Counter.#count; }
}

console.log(MathUtils.add(3, 4));       // 7
console.log(MathUtils.circleArea(5));   // 78.53...

new Counter(); new Counter(); new Counter();
console.log(Counter.getCount());        // 3